.travis.yml

```diff
@@ -2,33 +2,24 @@ language: php
 
 sudo: false
 
-php: [5.6, 7.0, 7.1, 7.2, 7.3]
+php: [5.6, 7.0, 7.1, 7.2, 7.3, 7.4]
 
 matrix:
   include:
-    - php: 5.4
```

Comment:
Please rollback removal of supported PHP versions.

We can always remove them once agreement is reached in https://github.com/minkphp/Mink/issues/787.